pegasus/migrations/83_create_channel_table_metadata.rb

```diff
@@ -0,0 +1,20 @@
+Sequel.migration do
+  up do
+    create_table?(:channel_table_metadata, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      foreign_key :app_id, index: true, null: false
+      foreign_key :storage_id, index: true
+      String :table_name, size: 50, index: true, null: false
+
+      Text :column_list
+
+      DateTime :updated_at, null: false
+
+      index [:channel_id,:storage_id,:table_name], unique: true
+    end
+  end
+
+  down do
+    drop_table?(:channel_table_metadata)
+  end
```

Comment:
I think we want to do this migration `unless [:staging, :adhoc, :test, :production].include?(rack_env)`, since we use DynamoDB in those environments. I know the app_tables and app_properties tables exist in all environments today, but I have an outstanding task (referenced [here](https://www.pivotaltracker.com/story/show/101145644)) to fix that.